BurnOutSharp/PackerType/InnoSetup.cs

```diff
@@ -13,10 +14,17 @@ public class InnoSetup : IContentCheck, IScannable
         /// </summary>
         private static readonly List<ContentMatchSet> contentMatchers = new List<ContentMatchSet>
         {
+            // Inno Setup Setup Data (
+            new ContentMatchSet(new byte?[]
+            {
+                0x49, 0x6E, 0x6E, 0x6F, 0x20, 0x53, 0x65, 0x74, 0x75, 0x70, 0x20, 0x53,
```

Comment:
Just to follow the style of the other checks, can you limit each of these to 8 bytes?
e.g the first line would be `0x49, 0x6E, 0x6E, 0x6F, 0x20, 0x53, 0x65, 0x74,`